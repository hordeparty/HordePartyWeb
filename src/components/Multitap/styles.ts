import styled from 'styled-components'

export const Wrapper = styled.div``

export const FrontTap = styled.div`
  width: 100%;
  border: 6px solid #000;
  height: 15rem;
  border-radius: 5px;
  background-color: #ccc;
  display: flex;
  flex-wrap: wrap;
  align-items: center;
  justify-content: space-evenly;
`
