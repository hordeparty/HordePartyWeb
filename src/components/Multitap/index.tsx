import Slot from 'components/Slot'
import * as S from './styles'

const Multitap = () => (
  <S.Wrapper>
    <S.FrontTap>
      <Slot />
      <Slot />
      <Slot />
      <Slot />
      <Slot />
    </S.FrontTap>
  </S.Wrapper>
)

export default Multitap
