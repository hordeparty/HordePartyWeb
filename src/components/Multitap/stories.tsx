import { Story, Meta } from '@storybook/react'
import Multitap from '.'

export default {
  title: 'Multitap',
  component: Multitap
} as Meta

export const Default: Story = () => <Multitap />
