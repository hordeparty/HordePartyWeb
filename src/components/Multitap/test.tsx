import { render } from '@testing-library/react'

import Multitap from '.'

describe('<Multitap />', () => {
  it('should render the Multitap component', () => {
    const { container } = render(<Multitap />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
