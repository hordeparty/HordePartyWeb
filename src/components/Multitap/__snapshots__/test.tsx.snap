// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Multitap /> should render the Multitap component 1`] = `
.c1 {
  width: 20rem;
  height: 3rem;
  border: 5px solid #000;
  border-radius: 5px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c1 > div:first-child {
  border-right: 5px solid #000;
  width: 15rem;
}

.c2 {
  width: 10rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: space-evenly;
  -webkit-justify-content: space-evenly;
  -ms-flex-pack: space-evenly;
  justify-content: space-evenly;
}

.c3 {
  width: 1rem;
  height: 1rem;
  border-radius: 50%;
  background-color: #000;
}

.c0 {
  width: 100%;
  border: 6px solid #000;
  height: 15rem;
  border-radius: 5px;
  background-color: #ccc;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: space-evenly;
  -webkit-justify-content: space-evenly;
  -ms-flex-pack: space-evenly;
  justify-content: space-evenly;
}

<div
  class=""
>
  <div
    class="c0"
  >
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <div
          class="c3"
        />
        <div
          class="c3"
        />
        <div
          class="c3"
        />
        <div
          class="c3"
        />
      </div>
      <div
        class="c2"
      >
        <div
          class="c3"
        />
        <div
          class="c3"
        />
        <div
          class="c3"
        />
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <div
          class="c3"
        />
        <div
          class="c3"
        />
        <div
          class="c3"
        />
        <div
          class="c3"
        />
      </div>
      <div
        class="c2"
      >
        <div
          class="c3"
        />
        <div
          class="c3"
        />
        <div
          class="c3"
        />
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <div
          class="c3"
        />
        <div
          class="c3"
        />
        <div
          class="c3"
        />
        <div
          class="c3"
        />
      </div>
      <div
        class="c2"
      >
        <div
          class="c3"
        />
        <div
          class="c3"
        />
        <div
          class="c3"
        />
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <div
          class="c3"
        />
        <div
          class="c3"
        />
        <div
          class="c3"
        />
        <div
          class="c3"
        />
      </div>
      <div
        class="c2"
      >
        <div
          class="c3"
        />
        <div
          class="c3"
        />
        <div
          class="c3"
        />
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <div
          class="c3"
        />
        <div
          class="c3"
        />
        <div
          class="c3"
        />
        <div
          class="c3"
        />
      </div>
      <div
        class="c2"
      >
        <div
          class="c3"
        />
        <div
          class="c3"
        />
        <div
          class="c3"
        />
      </div>
    </div>
  </div>
</div>
`
